src/lib.rs

```diff
@@ -181,6 +181,23 @@ impl<'a> Type<'a> {
         }
     }
 
+    /// Returns alignment of the type
+    pub fn align(&self) -> u64 {
+        match self {
+            Self::Byte => 1,
+            Self::Halfword => 2,
+            Self::Word | Self::Single => 4,
+            Self::Long | Self::Double => 8,
+            Self::Aggregate(td) => match td.align {
+                Some(align) => align,
+                None => {
+                    assert!(!td.items.is_empty(), "Invalid empty TypeDef");
```

Comment:
Is this really needed? Are typedefs required to have at least one item?

Reply:
Yes, absolutely. The only possible way is not-yet-supported opaque typedefs, but they require explicit alignment.